pkg/innerring/innerring.go

```diff
@@ -308,12 +308,17 @@ func (s *Server) registerCloser(f func() error) {
 	s.closers = append(s.closers, f)
 }
 
-// New creates instance of inner ring sever structure.
+// New creates instance of inner ring server structure.
 func New(ctx context.Context, log *zap.Logger, cfg *viper.Viper, errChan chan<- error) (*Server, error) {
 	var err error
 	server := &Server{log: log}
 
-	server.setHealthStatus(control.HealthStatus_HEALTH_STATUS_UNDEFINED)
+	server.setHealthStatus(control.HealthStatus_CREATE_SERVER)
```

Comment:
i think `HealthStatus_CREATE_SERVER` was not a problem at all. you started a server, it only created `&Server{log: log}`, so it is really undefined now. i guess the initial issue was mostly about extending codes: you set it like "STARTING_BLOCKCHAIN" and then call `server.bc.Run(ctx)`; you set it like "DEPLOYING_NETWORK" and then you call `deploy.Deploy(ctx, deployPrm)`, etc. it allows understand more about what is happening, starting IR may take minutes now, and all an admin got is minutes of "undefined" that goes immediately to READY then

Reply:
the rule of thumb may be: any operation that is _not_ about local CPU work but about any I/O that potentially may block and that an admin should know about

Reply:
Added new statuses that seemed to work with I/O and can bloc func, but I'm not sure about them.